assets/js/theme/common/utils/translations-utils.js

```diff
@@ -26,3 +26,27 @@ export const createTranslationDictionary = (context) => {
         return acc;
     }, {});
 };
+
+const defaultPageBuilderValues = {
+    pdp_sale_badge_label: 'On Sale!',
+    pdp_sold_out_label: 'Sold Out',
+    'pdp-sale-price-label': 'Now:',
+    'pdp-non-sale-price-label': 'Was:',
+    'pdp-retail-price-label': 'MSRP:',
+    'pdp-custom-fields-tab-label': 'Additional Information',
+};
+
+/**
+ * defines Translation for values from page builder (locally could be found in config.json)
+ */
+export const translatePageBuilderValues = () => {
+    $('[data-page-builder-key]').each((_, selector) => {
+        const $item = $(selector);
+        const itemText = $item.text().trim();
+        const itemDefaultTranslation = $item.data('default-translation');
+
+        if (itemText === defaultPageBuilderValues[$item.data('page-builder-key')] && itemText !== itemDefaultTranslation) {
```

Comment:
Maybe it will be better to change `itemDefaultTranslation` to `JsonTranslation`

Reply:
It's a taste thing, I prefer to use itemDefaultTranslation